client/src/util/metrics/helpers.js

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
are these only helpers for `metrics` but maybe a general `ExtensionElementHelper`?

Reply:
Where would you put them?

Reply:
Maybe just one directory up in `client/src/util`, maybe just as `ModelUtil`.